lib/entities/QuoteResponse.ts

```diff
@@ -43,10 +47,27 @@ export class QuoteResponse implements QuoteResponseData {
   }
 
   public static fromRFQ(request: QuoteRequestData, data: RfqResponse, type: TradeType): ValidatedResponse {
+    let validationError: ValidationError | undefined;
+
     const responseValidation = RfqResponseJoi.validate(data, {
```

Comment:
would be ideal to do the token addr check in here. Maybe a variant of https://github.com/Uniswap/uniswapx-parameterization-api/blob/4ba4e10dab3f170c06be2ddfa584874bb5e8b8ce/lib/handlers/quote/schema.ts#L8 ?